### DISCUSSION TITLE
A stupid Question: which develop tool should be used when modifying O3DE

### DISCUSSION THREAD

**Discussion by dnqbob** [o3de/o3de#6501] [Help Wanted]
I tried to use Visual Studio but I cannot find any clue on where is the project file. So which tool to use and how to open O3DE in project?

**Comment by soltsmank** [comment]
Hi,

I think you have to make it by running cmake. Follow the guide:
https://o3de.org/docs/welcome-guide/setup/setup-from-github/building-windows/
(There is an article for linux as well)
